cw-deb.mk

```diff
@@ -108,19 +83,19 @@ deb-build:
 		echo "  * build from revision $$(git rev-parse HEAD)" >>debian/changelog;\
 	fi
 	echo " -- $(CW_SIGNER_REAL) <$(CW_SIGNER)>  $$(date -R)" >>debian/changelog
-ifneq ($(wildcard $(LICENSE)),)
-	if [ -e debian/copyright.repo ]; then\
-		cp debian/copyright.repo debian/copyright;\
-		echo "" >> debian/copyright;\
-	        echo "Files: *" >> debian/copyright;\
-	        echo "Copyright: Metaswitch Networks" >> debian/copyright;\
-	        echo "License: GPL-3+ with OpenSSL exception" >> debian/copyright;\
-		cat $(LICENSE) >> debian/copyright;\
-	fi
+
+# Construct a Debian Copyright file.   If there is a COPYRIGHT file provided
```

Comment:
I strongly dislike your triple spaces. Even double is unnecessary, but this is   madness :p